optional_plugins/varianter_cit/tests/test_unit.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+
+from avocado_varianter_cit import Cit
+
+
+class CitTest(unittest.TestCase):
+
+    PARAMS = [('key1', ['x1', 'x2', 'x3']), ('key2', ['y1', 'y2', 'y3']),
+              ('key3', ['z1', 'z2', 'z3']), ('key4', ['w1', 'w2', 'w3'])]
+
+    def test_orders(self):
+        for i in range(len(self.PARAMS)):
+            cit = Cit(self.PARAMS, i+1)
+            headers, _ = cit.combine()
+            # headers should always be the same, no matter the order
+            # of combinations
+            self.assertEqual(headers, ['key1', 'key2', 'key3', 'key4'])
```

Comment:
perhaps you could also check that with more DOC the more variants are produced (or more_or_equal?)